### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by AkintoyeOlamide**
In this milestone, I connected the CREATE_BOOK and REMOVE_BOOK
actions with the React component and make the page dynamic.

### CODE REVIEW

File: src/containers/BookList.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
-/* eslint-disable */
```
Comment (rindrajosia) [inline src/containers/BookList.js:1]: - [x] [REQUIRED] Please don't disable eslint. It is like making some changes in the linter file.

File: src/containers/BookList.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-const mapDispatchToProps = (dispatch) => ({
-  removeBook: (ID) => {
+const mapDispatchToProps = dispatch => ({
+  removeBook: book => {
     dispatch({
-      ID,
-      type: "REMOVE_BOOK",
+      ID: book.ID,
+      type: 'REMOVE_BOOK',
     });
```
Comment (rindrajosia) [inline src/containers/BookList.js:11]: - [ ] [REQUIRED] it connects the REMOVE_BOOK action. In fact, you have already a REMOVE_BOOK action inside `src/actions/index.js` so, You don't need to redefine it.

File: src/containers/BooksForm.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import React, { useState } from 'react';
+/*eslint-disable */
```
Comment (rindrajosia) [inline src/containers/BooksForm.js:1]: - [x] [REQUIRED] Please don't disable eslint. It is like making some changes in the linter file.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] I got an error related to your prop-types when I open your APP on my browser, please fix it.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Hey, good job making the requested changes. 
I ran your app locally and got this error: 
Screenshot from 2021-08-06 13-38-22
Fix and send another request. Good luck


### Martin Afani
#### Slack: whiz25

File: src/containers/BooksForm.js (hunk 3 above)
Comment (rindrajosia) [inline src/containers/BooksForm.js:1]: - [ ] [REQUIRED] Please `don't disable eslint`. It is like making changes in the linter config.

File: src/containers/BookList.js (hunk 2 above)
Comment (rindrajosia) [reply src/containers/BookList.js:11]: It will be nice if you work on it

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] [REQUIRED] The error related to the prop-types reappears. Please fix it:


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Fix the linter errors and submit for another review. No PR can be approved when linters checks are failing.


### Martin Afani
#### Slack: whiz25

**Review by whiz25 — APPROVED** [review]
## Approved 👍🏿 👍🏿 👍🏿 
Nice work making the requested changes. All linter errors have been fixed.


### Martin Afani
#### Slack: whiz25

**Pull request merged**